install_files/ansible-base/roles/grsecurity/tasks/main.yml

```diff
@@ -9,11 +9,6 @@
   tags:
     - grsec
 
-- include: from_local_pkg_install_grsec.yml
```

Comment:
Since this task is no longer valid, the installation of securedrop-grsec from the apt repo in `.../grsecurity/tasks/from_fpf_repo_install_grsec.yml` should probably always happen in both staging and prod. (Staging is currently affected by this but prod installs would not be a problem.)